src/app/routes/meditatedRecently.js

```diff
@@ -0,0 +1,14 @@
+export default (user) => {
+	if (!user || !'lastMeditation' in user || !user.lastMeditation instanceof Date) {
```

Comment:
And make sure to use two spaces instead of four.
A comment for the file would be nice, too :).